specifyweb/frontend/js_src/lib/components/Merging/Usages.tsx

```diff
@@ -16,32 +16,56 @@ export function UsagesSection({
 }: {
   readonly resources: RA<SpecifyResource<AnySchema>>;
 }): JSX.Element {
+  const [blockerLoaded, setBlockerLoaded] = React.useState<boolean>(false);
+  const handleBlockersLoaded = React.useCallback(
+    () => setBlockerLoaded(true),
+    [setBlockerLoaded]
+  );
   return (
-    <MergeRow header={mergingText.referencesToRecord()}>
-      <td>{commonText.notApplicable()}</td>
+    <MergeRow className="!items-start" header={mergingText.linkedRecords()}>
+      <td className="!items-start">{commonText.notApplicable()}</td>
       {resources.map((resource, index) => (
-        <Usages key={index} resource={resource} />
+        <Usages
+          blockersLoaded={blockerLoaded}
+          key={index}
+          resource={resource}
+          onBlockersLoaded={handleBlockersLoaded}
+        />
       ))}
     </MergeRow>
   );
 }
 
 function Usages({
   resource,
+  blockersLoaded,
+  onBlockersLoaded: handleBlockersLoaded,
 }: {
+  readonly blockersLoaded: boolean;
   readonly resource: SpecifyResource<AnySchema>;
+  readonly onBlockersLoaded: () => void;
 }): JSX.Element {
   const [loadBlockers, setLoadBlockers] = React.useState<boolean>(false);
 
   const [blockers, setBlockers] = useAsyncState(
     React.useCallback(
-      async () => (loadBlockers ? fetchBlockers(resource, true) : undefined),
-      [loadBlockers, resource]
+      async () =>
+        loadBlockers
+          ? fetchBlockers(resource, true).then((data) => {
+              if (data.length > 0) handleBlockersLoaded();
+              return data;
+            })
+          : undefined,
+      [loadBlockers, resource, handleBlockersLoaded]
     ),
     false
   );
   return (
-    <td className="h-[theme(spacing.40)] flex-col !items-start overflow-auto">
+    <td
+      className={`${
+        blockersLoaded ? 'h-[theme(spacing.40)]' : 'h-[theme(spacing.14)]'
+      } flex-col !items-start overflow-auto`}
```

Comment:
you can make this more readable by formatting it like this:

```suggestion
      className={`
        flex-col !items-start overflow-auto
        ${blockersLoaded ? 'h-[theme(spacing.40)]' : 'h-[theme(spacing.14)]'}
      `}
```